uw-frame-components/portal/misc/filters.js

```diff
@@ -65,6 +65,15 @@ define(['angular'], function(angular) {
         return filtered;
       };
     })
+    .filter('urlToTarget', function() {
+      return function(url) {
+        var result = '_self';
+        if (url && -1 < url.indexOf('//')) {
```

Comment:
:question: can this ever be true?
[`indexOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf) will return -1 if not found and a number >= 0 if value is found. It will never be less than -1 AFAIK.

Reply:
I think so...
not found: -1 < -1 (false)
found: -1 < 0 or above (true)

Reply:
yep, my bad I misread that.
:art: avoid [Yoda Conditions](https://en.wikipedia.org/wiki/Yoda_conditions)

Reply:
Please express your concerns and reservations about code style and design choices, but I'd appreciate something more than an "avoid ______" statement and a link to Wikipedia.

Reply:
Okay, I usually use the 🎨 emoji has shorthand to say this it is a code style preference.
If you prefer long hand

> It is my personal preference that we should avoid [Yoda Conditions](https://en.wikipedia.org/wiki/Yoda_conditions) in the code style